agent/app/agent.go

```diff
@@ -334,9 +338,31 @@ func (agent *ecsAgent) doStart(containerChangeEventStream *eventstream.EventStre
 		StringSetValue: aws.StringSlice(gpuIDs),
 	}
 
+	daemonDefinitions, err := md.ImportAll()
+	// we will not panic when daemons fail to import; agent should continue running
+	// container instance health will act as the mechanism to advertise daemon failures
+	if err != nil {
+		seelog.Errorf("Daemon import mountpoint failure: %s", err)
+	}
+	if daemonDefinitions == nil || len(daemonDefinitions) == 0 {
+		seelog.Infof("daemonDefinitions is empty/nil after import")
+	}
+	// load and add daemons to agent
+	for _, md := range daemonDefinitions {
+		thisDaemon := dm.NewDaemonManager(md)
+		if _, err := thisDaemon.LoadImage(agent.ctx, agent.dockerClient); err != nil {
+			seelog.Errorf("Managed Daemon Load failure %v", err)
+		}
+		if loaded, _ := thisDaemon.IsLoaded(agent.dockerClient); loaded {
```

Comment:
Is it safe to proceed with `thisDaemon` if `LoadImage` failed? It's not clear to me what `IsLoaded` does.